#
# (c) Copyright 2015-2016 River Systems, all rights reserved.
#
# This is proprietary software, unauthorized distribution is not permitted.
#

string label

uint16 xi
uint16 yi

uint16 xf
uint16 yf

srslib_framework/MapNote note